Add new ComboBox features and update articles. (#642)
telerik/maui-docs: controls/combobox/configuration.md

![ComboBox Placeholder](images/combobox-visual-structure.png)

Here is an example of setting the `PlaceholderColor` property:

<snippet id='combobox-configuration-placeholder-color'/>

> and the result:

Reviewer: Use complete sentences.

> ![ComboBox Placeholder Color](images/combobox-placeholder-color.png)

Reviewer: ```suggestion
![.NET MAUI ComboBox Placeholder Color](images/combobox-placeholder-color.png)
```

* `DropDownWidth`(`double`)&mdash;Defines the width of the dropdown of the control.
* `DropDownHeight`(`double`)&mdash;Defines the height of the dropdown of the control.
* `DropDownVerticalOffset`(`double`)&mdash;Defines the vertical offset of the drop down part of the control. This property allows an option to modify the control with no space between the ComboBox and the dropdown.
* `IsDropDownOpen`(`bool`)&mdash;Defines whether the dropdown part of the control is opened. The default value is `true`. 
* `IsDropdownClosedOnSelection`(`bool`)&mdash;Defines whether the dropdown will be closed when the item is selected or deselected. The default value is `true`.

> Here is an example with `DropDownWidth` property set:

Reviewer: ```suggestion
Here is an example that uses the `DropDownWidth` property:
```

<snippet id='combobox-configuration-dropdownwidth'/>

> Here is an example with `DropDownHeight` property set:

Reviewer: ```suggestion
Here is an example that uses the `DropDownHeight` property:
```

<snippet id='combobox-configuration-dropdownheight'/>

> Here is an example with `DropDownVerticalOffset` property set:

Reviewer: ```suggestion
Here is an example that uses the `DropDownVerticalOffset` property:
```
